models/enterprise_sonic/snmp/merged_example_01.txt

```diff
@@ -1,31 +1,30 @@
 # Using merged
-
-# Before state:
-# -------------
 #
-#rno-dctor-1ar01c01sw02# show snmp-server community
+# Before state:
+#---------------
 #
-#         Community Name                     Group Name
-#-------------------------------- --------------------------------
-#private                          ro
-#public                           ro
+# show running-configuration | grep snmp
+# (no snmp-server configuration present)
 #
 
-- name: Merge a single SNMP community
+- name: Merge specific option snmp-server configuration
   sonic_snmp:
-    config:
-      - community: "public123"
-        access: "ro"
-    state: merged
+      config:
+          user:
+              - name: user1
+                auth:
+                  auth_type: md5
+                  key: U2FsdGVkX18J+L+L9OyQYWpAkGUrTgcg/6xzZoDjCbQw1ISHJ5mxmxrYZgQypEUXDeNe6rBupsc9sVDJBKxrwA==
+                priv:
+                  aes_key: aes-128
+                encrypted: False
```

Comment:
Use lower case for YAML "truthy" values. I this case, use "false" instead of "False".